pysearch/templates/about.jinja2

```diff
@@ -0,0 +1,66 @@
+{% extends "layout.jinja2" %}
+
+{% block nav %}
+
+<nav class="navbar ">
+  <div class="container-fluid">
+    <div class="collapse navbar-collapse" id="myNavbar">
+
+    <div class="col-lg-9"></div>
+      <ul class="nav navbar-nav topBar">
+        <li class="active"><a href="/">Home</a></li>
+        <li><a href="/about">About</a></li>
+      </ul>
+    <div class="col-lg-2"></div>
+
+    </div>
+  </div>
+</nav>
+{%endblock%}
+
+
+{% block body %}
+<div class="row">
+<div class="col-sm-1"></div>
+  <div class="col-sm-10">
+    <div class="panel panel-primary">
+      <div class="panel-heading">PySearch</div>
+      <div class="panel-body"></div>
+      <div class="panel-footer">PySearch is a one of a kind search engine. Unlike a conventional search engine, input a URL and find similar sites. Sites are analyzed and ranked using a unique matching algorithm.</div>
+    </div>
+  </div>
+</div>
+<div class="container">
+  <div class="row">
+    <div class="col-sm-4">
+      <div class="panel panel-primary">
+        <div class="panel-heading">Marc Fieser</div>
+        <div class="panel-body"><img src="../static/marc.jpg" class="img-responsive" style="width:100%" alt="Image"></div>
```

Comment:
If this is a static file, why not use `request.static_url` or `request.static_path` to retrieve it?